src/base/static/components/form-fields/rich-textarea-field.scss

```diff
@@ -0,0 +1,25 @@
+.mapseed-rich-textarea-field {
```

Comment:
I don't think it's necessary to be nesting our class names. It seems simpler to have a single class name per block/element, instead of mapping a combination of multiple class names to see which CSS block is associated with which JSX element.